src/flint/flint_base/flint_base.pyx

```diff
@@ -27,6 +27,14 @@ cdef class flint_poly(flint_elem):
             yield self[i]
 
     def coeffs(self):
+        """
+        Returns the coefficients of ``self`` as a list
+
+            >>> from flint import fmpz_poly
+            >>> f = fmpz_poly([1,2,3,4,5])
+            >>> f.coeffs()
+            [1,2,3,4,5]
```

Comment:
This module needs to be added to the doctests:
```diff
diff --git a/src/flint/test/__main__.py b/src/flint/test/__main__.py
index 517eb15..c22e5f4 100644
--- a/src/flint/test/__main__.py
+++ b/src/flint/test/__main__.py
@@ -52,6 +52,8 @@ def run_doctests(verbose=None):
     """Run the python-flint doctests"""
     # Here verbose=True shows a lot of output.
     modules = [flint.pyflint,
+               flint.flint_base.flint_base,
+               flint.flint_base.flint_context,
                flint.types.fmpz,
                flint.types.fmpz_poly,
                flint.types.fmpz_mat,
```
The doctest here fails:
```
File "/home/oscar/current/active/python-flint/src/flint/flint_base/flint_base.cpython-311-x86_64-linux-gnu.so", line ?, in flint.flint_base.flint_base.__test__.flint_poly.coeffs (line 29)
Failed example:
    f.coeffs()
Expected:
    [1,2,3,4,5]
Got:
    [1, 2, 3, 4, 5]
```

Reply:
Ahh thanks! I'll fix this.